### DISCUSSION TITLE
Mass weighted covariance analysis

### DISCUSSION THREAD

**Discussion by Sanjoy-Paul** [prody/ProDy#1885] [Archived Issues]
Hi,
    I am interested in calculating schlitter entropy and quasiharmonic entropy for a .dcd file having 5000 no. of frames.For this mass weighted covariance matrix must be developed.Is it possible to produce such mass weighted variance-covariance matrix and mass weighted eigenvalues from a .dcd file in prody ?

**Comment by jamesmkrieger** [comment]
I realised my last comment was wrong. We will look into this more another time.
